content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+It is also possible to enable and disable plugins using the <<cli#,Jenkins CLI>>.
+Since Jenkins 2.136, administrators may invoke the `enable-plugin` command to 
```

Comment:
NIT: add link anchor to changelog entry for this core version?

Reply:
Done!